### ISSUE TITLE
New line characters embedded in fields get removed

### ISSUE THREAD

**Issue by HeyRatFans** [thephpleague/csv#124]
As discovered while fixing up pull request #121, when using `Reader` to read a CSV with Linux-style new-lines embedded in fields, the new-lines are erroneously removed. For example, given the data `1, 'two', 3, "new\nline"`, this becomes `1, 'two', 3, "newline"`.

When using PHP's native fopen and fgetcsv, the new lines remain present in the data read from the file.

This issue affects release 7.1.2 of thephpleague/csv and is known to be present on both PHP 5.6.10 on Ubuntu 14.04 and 5.5.25 on Windows 7.

**Comment by nyamsprod** [comment]
Ok... I think I know the source of this bug but alas it will be fix in v8.0.0 because it will cause BC break otherwise. The problem is by default League\CSv uses the `SplFileObject::DROP_NEW_LINE` constant instead of the `SplFileObject::SKIP_EMPTY` one. changing this default settings affect many usage of the package. 

This issue is partially solved with issue #99 

Here's a simple workaround

```php
$csv = Reader::createFromPath('path/to/file.csv');
$csv->setFlags(SplFileObject::READ_AHEAD|SplFileObject::SKIP_EMPTY);
```
This is one reason I want to completely remove the SPL flags in version 8 see issue #105

**Comment by nyamsprod** [comment]
This bug is fixed in the master and will be part of the next release version
